exercises/proverb/.meta/src/reference/java/Proverb.java

```diff
@@ -0,0 +1,28 @@
+final class Proverb {
+    private String[] words;
+    private String initWord, qualifier, result = "";
+
+
+    public Proverb(String... words) {
+        this.words = words;
+    }
+
+    public void setQualifier(String qualifier) {
```

Comment:
This is an example of where the package-private access level could be used - we try to [limit scope](https://github.com/exercism/java/blob/master/POLICIES.md#adhere-to-best-practices).

Reply:
Thanks for the feedback. I really appreciate it. I'm relatively new to Java and am looking at the docs right now. When you say limiting scope by using package-private access level, are you referring to removing the `public` ahead of the method names? For example `public void setQualifier(......)` would simply become `void setQualifier(.....)`. This means that the method (or whatever member) can only be accessed within its own package. If that's correct, I'll go ahead and push up those changes.

Reply:
That's exactly right - I think this is a good change, as nothing outside of the package requires access to those files, and generally it is best practise to limit scope 🙂

Reply:
Perfect. Thank you so much. Exercism.io is one of the greatest resources when learning a new language.